BUILD

```diff
@@ -59,4 +124,44 @@ sh_test(
         "tests",
          "clang+llvm-16.0.4-x86_64-linux-gnu-ubuntu-22.04",
     ],
-)
\ No newline at end of file
+)
+
+# This build rule is to run the series of tests defined in 
+# wasmtestreport.py
+py_binary(
+    name = "python_tests",
+    srcs = ["wasmtestreport.py"],
+    main = "wasmtestreport.py",    
+    # This ensures the tests have access to the folders required.
+    # The logs from the previous steps are included to ensure 
+    # the rules that create them are run.  This is a requirement
+    # to use a genrule as a dependency.
+    data = [
+        "tests",
+         #"clang+llvm-16.0.4-x86_64-linux-gnu-ubuntu-22.04",
```

Comment:
why is this commented out?

Reply:
It should be removed.  It got replaced by the imported filegroup clang_files

Reply:
I've taken it out and next push will mark it removed